packages/components/app/styles/components/modal.scss

```diff
@@ -120,7 +85,7 @@
     border-bottom: 1px solid var(--token-color-border-warning);
   }
 
-  .hds-modal__tagline {
+  .hds-dialog-primitive__tagline {
```

Comment:
A reminder to discuss later if it's better at that point also change the sibling declaration to override the `hds-dialog-primitive__header` instead of the `hds-modal__header`.

Reply:
I can change it if you think that would be better